wpimath/src/main/native/include/frc/util/PIDConstants.h

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+#pragma once
+
+#include "units/time.h"
+
+namespace frc {
+/** A class to configure a PIDController with */
+class PIDConstants {
```

Comment:
This makes more sense as a struct.

Reply:
I chose to create it as a class so we can check if the values are valid on initialisation easily

Reply:
structs can have constructors too.

Reply:
That's crazy I'll make it a struct

Reply:
In C++, the only difference between classes and structs is that structs are public by default. 

https://learn.microsoft.com/en-us/cpp/cpp/classes-and-structs-cpp?view=msvc-170

Everything that is valid for a class, is also valid for a struct.